Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/FareProduct.hs

```diff
@@ -65,6 +65,9 @@ findByMerchantOpCityIdVariantArea (Id merchantOpCityId) tripCategory vehicleVari
         ]
     ]
 
+findAllFareProductByMerchantOpCityId :: (MonadFlow m, EsqDBFlow m r, CacheFlow m r) => Id DMOC.MerchantOperatingCity -> m [Domain.FareProduct]
+findAllFareProductByMerchantOpCityId (Id merchantOpCityId) = findAllWithKV [Se.Is BeamFP.merchantOperatingCityId $ Se.Eq merchantOpCityId]
```

Comment:
findBy merchantOperatingCityId, area = Default, merchantId

Reply:
@khuzema786 bro can you elaborate?